package.json

```diff
@@ -1,20 +1,28 @@
 {
   "name": "@tduniec/backstage-timesaver-plugin",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
each package has its versions in plugin/*/package.json

Reply:
Hi @tduniec. Yes, in turn I bumped up the versions for all the plugins that were modified during this feature. Moreover, I added linting and formatting functionalities based on Backstage configurations, which in turn needed some dependencies, hence modifying the root package. Is it ok if this version bumps up or would you like to keep it at 1.0.0 ?

Reply:
it is ok to bump it :)

Reply:
Nice!